packages/expo-doctor/src/checks/InstalledDependencyVersionCheck.ts

```diff
@@ -35,7 +33,9 @@ export class InstalledDependencyVersionCheck implements DoctorCheck {
     return {
       isSuccessful: issues.length === 0,
       issues,
-      advice,
+      advice: issues.length
+        ? `Use npx expo install --check to review and upgrade your dependencies.`
```

Comment:
```suggestion
        ? `Use 'npx expo install --check' to review and upgrade your dependencies.`
```

maybe worth wrapping in quotes to separate the command out from the rest of the text?